hack/generate-release-notes.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
Want to put this in `hack/boilerplate/boilerplate.sh.txt`?

Reply:
done https://github.com/knative/serving/pull/9604